pixart.js

```diff
@@ -1 +1,37 @@
+var button = $("#button");
+// var input = $(".text");
+var brush = $(".brush");
 
+function changeBrushColortoInput () {
+  var color = document.getElementById("color-field").value;
+  // var color = $("#color-field").val(); //need to get the value from form
+  $(".brush").css("background","color");
+
+}
+
+button.on("click", changeBrushColortoInput());
```

Comment:
When passing a callback function into an event listener, it should never be a function invokation (i.e., a function with `( )` at the end of it). If we do this, the function will be run immediately upon page load instead of when the event listener is triggered.

Instead, it should be a function reference (i.e., a function without `( )` at the end of it). A function reference contains a function definition. By passing a function definition to an event listener, we're giving it a function that it can later on whenever the listener is triggered.